company-profiles/tech-angels.md

```diff
@@ -0,0 +1,43 @@
+# Tech-Angels
+
+## Company blurb
+
+Tech-Angels is a Services and Software company. 
+We excel in web and software development, and creating innovating services using Go (http://golang.org) + Angularjs.
+
+Tech-Angels features 3 kind of activities:
+
+* Consulting (engineering, code, audits, etc.)
+* Hosting 
+* SaaS
+
+We're the guys behing https://gemnasium.com
+
+## Company size
+
+We want to keep the company small, to remain experts in what we do. 
+We are currently less than 10, and will grow to 20 in the future.
+
+## Remote status
+
+Tech-Angels doesn't have any office. We are all working remotly, except the ones behing on clients premises.
+We use online tools to synchronize.
+
+## Region
+
+We're based in Paris (France), Miami (FL, USA), and Québec City (Canada)
+
+## Company technologies
+
+We are coming from the Ruby On Rails world. 
+While we have been doing a lot of development in the 5 first years of the company, we have switched
+recently to [Go](http://golang.org) and [Angular.js](https://angularjs.org/).
+We are relying on [openshift](https://github.com/openshift/origin) (/docker) and vsphere for our infrastructure, and [PostgreSQL](http://postgresql.org). 
+
+## Office Locations
+
+Our main office is in Paris (France) and we have a subsidiary in Florida.
+
+## How to apply
+
+Send your application to am9ic0B0ZWNoLWFuZ2Vscy5jb20=
```

Comment:
Yes, that an encoded email address. Applicants will figure out how to decode it ;)